lucien-ios/Protocols/CameraViewDelegate.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  CameraViewDelegate.swift
+//  lucien-ios
+//
+//  Created by Ahmed, Guled on 10/6/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import Foundation
+
+protocol CameraViewDelegate: class {
+    func getImage(image: UIImage)
```

Comment:
Naming here is a little off - take a look at `UITableViewDelegate` and others for the format. It's usually along the lines of `<view controller>Did<do action>`. In this case it should be something like `cameraViewController(viewController: CameraViewController, didCapture image: UIImage)`. Because the delegate never calls its own methods, the methods should be named like callbacks, not like actions.